### PR TITLE
doc/release-notes: jewel updates

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: doc/release-notes.rst [hunk 1]
```diff
@@ -220,9 +220,7 @@ Major Changes from Infernalis
     feature.  The plan is for it to become the default backend in the
     K or L release.
   * The OSD now persists scrub results and provides a librados to
```
Comment (jdurgin) [inline doc/release-notes.rst]: 'provides a librados *api* to'?

Comment (tchaikov) [inline doc/release-notes.rst]: or "provides librados APIs to"

**Comment by tchaikov** [comment]
needs rebase.

**Comment by smithfarm** [comment]
@liewegas please see https://github.com/liewegas/ceph/pull/7

**Pull request merged**